src/pages/Transaction/TransactionComp.tsx

```diff
@@ -26,8 +29,13 @@ import SimpleButton from '../../components/SimpleButton'
 import HashTag from '../../components/HashTag'
 import { useAddrFormatToggle } from '../../utils/hook'
 import ComparedToMaxTooltip from '../../components/Tooltip/ComparedToMaxTooltip'
+import styles from './styles.module.scss'
+import { defaultTransactionLiteDetails } from './state'
+import { fetchTransactionLiteDetailsByHash } from '../../service/http/fetcher'
+import { WithdrawPopoverInfo, isDaoWithdrawCell } from '../../components/TransactionItem/TransactionItemCell'
 
 const showTxStatus = (txStatus: string) => txStatus.replace(/^\S/, s => s.toUpperCase())
+const defaultLayout = 'professional'
```

Comment:
This field has been used on several pages, and will be used on more pages along with the improvement of UX, so it can be defined in https://github.com/Magickbase/ckb-explorer-frontend/tree/develop/src/constants